packages/collector/test/tracing/cloud/aws-sdk/v2/sqs/receiveMessage.js

```diff
@@ -5,7 +5,12 @@
 
 'use strict';
 
-const instana = require('../../../../../..')();
+const instana = require('../../../../../..')({
+  tracing: {
+    // aws sdk produces some fs calls to read the config from disk on instantiation which we ignore.
+    useOpentelemetry: false
```

Comment:
_question:_ do we stil need to exclude this explicitly via config? IIRC you found a solution for not capturing fs calls outside of an ongoing tracing and the AWS SDK instantiation should happen outside of any request processing, wouldn't it?

Reply:
I was able to remove it for dynamodb tests, but not for sqs. Added a comment why.